vapi/namespace/simulator/simulator.go

```diff
@@ -281,7 +281,6 @@ func (h *Handler) namespaces(w http.ResponseWriter, r *http.Request) {
 		if len(subpath) > 0 {
 			if entry, contains := namespacesMap[subpath]; contains {
 				var spec namespace.NamespacesInstanceUpdateSpec
-				// If vapi.Decode fails it sets the status to bad request
```

Comment:
note: removed these comments since it's included in the function doc (and we use vapi.Decode this way all over)